### DISCUSSION TITLE
Expandable Blocks

### DISCUSSION THREAD

**Discussion by iced-queen** [GitbookIO/community#164] [Feature Requests]
### What's your feature/idea?

My suggestion is to allow expandable blocks inside other expandable blocks.
That way it's much easier to create categories inside exbandable blocks.

Maybe in general allow other types of blocks inside other expandable blocks.
So e.g. it'd also be possible to have Info Blocks inside Expandable Blocks.

### What's your desired outcome?

To have this feature implemented. For my project at least, it'd look so much cleaner if I could have expandable blocks inside other expandable blocks to sort of create categories.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by aikoii** [comment]
Hey @chocobearxo,

Thanks for your suggestion and I appreciate you explaining how it would help you out.

A similar feature request was captured here: https://github.com/orgs/GitbookIO/discussions/16 🙂.
 
I'll be closing this request in favor of the existing one.
